README.md

```diff
@@ -4,28 +4,28 @@ Dune (Jbuilder) - A composable build system
 __Jbuilder has been renamed to Dune__. A full renaming of the documentation and
```

Comment:
we should probably adapt this sentence

Reply:
Yes, and maybe make the "dune (jbuilder)" part less prominent as well.

Reply:
agreed, we can remove jbuilder from the title